takoyaki/src/main/java/frc/robot/Constants.java

```diff
@@ -4,12 +4,21 @@
 
 package frc.robot;
 
-/**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
- * globally (i.e. public static). Do not put anything functional in this class.
- *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
- * constants are needed, to reduce verbosity.
- */
-public final class Constants {}
+import edu.wpi.first.wpilibj.XboxController;
+
+public final class Constants {
+    public static int
+        climbUp = XboxController.Button.kY.value,
+        climbDown = XboxController.Button.kA.value,
+        climbBrakeOn = XboxController.Button.kX.value,
+        climbBrakeOff = XboxController.Button.kB.value,
+        
+        climbSolenoidChannel = 1,
+        climbLeftCAN = 15,
```

Comment:
we generally name the constants LEFT_MOTOR_ID

Reply:
Or `LEFT_DRIVE_MOTOR_CAN_ID`. Tells left motor on what subsystem and what type of ID.